driver/src/withdraw_driver.rs

```diff
@@ -57,11 +57,11 @@ pub fn run_withdraw_listener<D, C>(db: &D, contract: &C) -> Result<(bool), Drive
             let withdrawal_merkle_root = withdraws.root_hash(&valid_withdraws);
             let new_state_root = updated_balances.rolling_hash(balances.state_index + 1);
             
-            println!("New State_hash is {}, Valid Withdraw Merkle Root is {}", new_state_root, withdrawal_merkle_root);
+            debug!("New State_hash is {}, Valid Withdraw Merkle Root is {}", new_state_root, withdrawal_merkle_root);
             contract.apply_withdraws(slot, withdrawal_merkle_root, state_root, new_state_root, contract_withdraw_hash)?;
             return Ok(true);
         } else {
-            println!("Need to wait before processing withdraw_slot {:?}", slot);
+            info!("Need to wait before processing withdraw_slot {:?}", slot);
```

Comment:
I see only one `info` and I believe that many of the `debug` above should also be `info`.

Reply:
You are right. I inversed the "severeness" of DEBUG and INFO in my head.

Reply:
Killer!